### PR TITLE
m.10000recipe.com - Ad

### PR DESCRIPTION
**Pull request by green1052**

### CODE REVIEW

File: filters-share/specific_ELEMHIDE.txt [hunk 1]
```diff
@@ -22,7 +22,9 @@ dcinside.com##div[class$="banner"]
 toomics.com,nongmin.com,ilbe.com##div[class$="-banner"]
 ! ##div[class^="banner"]
 m.edaily.co.kr,ichannela.com,m.news1.kr##div[class^="banner"]
-! 
+!
+m.10000recipe.com##.view_sp_banner
+m.10000recipe.com###_fixedBannerAppInstall
```
Comment (piquark6046) [inline filters-share/specific_ELEMHIDE.txt]: List-KR does not block the app notification.
https://github.com/AdguardTeam/AdguardFilters/pull/118593

File: filters-share/specific_ELEMHIDE.txt [hunk 2]
```diff
@@ -22,7 +22,9 @@ dcinside.com##div[class$="banner"]
 toomics.com,nongmin.com,ilbe.com##div[class$="-banner"]
 ! ##div[class^="banner"]
 m.edaily.co.kr,ichannela.com,m.news1.kr##div[class^="banner"]
-! 
+!
+m.10000recipe.com##.view_sp_banner
```
Comment (piquark6046) [inline filters-share/specific_ELEMHIDE.txt]: Can you provide the URL?
If it is private, please email me.

**Review by piquark6046 — CHANGES_REQUESTED** [review]

File: filters-share/specific_ELEMHIDE.txt (hunk 2 above)
Comment (green1052) [reply filters-share/specific_ELEMHIDE.txt]: https://m.10000recipe.com/recipe/step.html?seq=6937381&step=0

File: filters-share/specific_ELEMHIDE.txt (hunk 1 above)
Comment (green1052) [reply filters-share/specific_ELEMHIDE.txt]: ok

**Comment by QiAnCeNo** [comment]
This also block there own contest.

**Comment by piquark6046** [comment]
@QiAnCeNo 
URL?

**Comment by seia-soto** [comment]
It actually contains their own content but we can completely decide what is the ad from the link urls.

```css
##.view_sp_banner a[href*="event.html?"] > img[src*="recipe1.ezmember.co.kr"]
```

**Comment by seia-soto** [comment]
Or we can ensure the user experience by removing the ad element completely like:
```css
m.10000recipe.com##.view_sp_banner div:has(> a[href*="event.html?"] > img[src*="recipe1.ezmember.co.kr/cache/shop"]):remove()
```

**Comment by piquark6046** [comment]
@seia-soto Can you give me an URL?

**Comment by seia-soto** [comment]
> @seia-soto Can you give me an URL?

I just saw the URL from green: https://m.10000recipe.com/recipe/step.html?seq=6937381&step=0

There is banner that advertising the competition held on 10krecipe (the website itself).

(image)

image

**Comment by seia-soto** [comment]
I recognize it is annoying because the size of the banner is too big but it should be go to AdGuardFilters as we did always.

https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/sections/annoyances.txt

**Review by piquark6046 — APPROVED** [review]

**Review by seia-soto — APPROVED** [review]

**Comment by seia-soto** [comment]
@green1052 Will you make a PR on AdGuardFilters? otherwise I'll.

> Got a respond from @green1052 , "You do it, ng"

**Pull request merged**